use-cases/call-center/src/main/java/org/acme/callcenter/data/DataGenerator.java

```diff
@@ -41,7 +42,8 @@ private static Set<Skill> buildSkillSet(Skill... skills) {
     }
 
     public CallCenter generateCallCenter() {
-        return new CallCenter(EnumSet.allOf(Skill.class), Arrays.asList(AGENTS), new ArrayList<>());
+        return new CallCenter(EnumSet.allOf(Skill.class), Arrays.stream(AGENTS).map(Agent::copy).collect(Collectors.toList()),
```

Comment:
If the intention is to avoid reusing the same `Agent` instances, I suggest creating them in this method directly and removing the static array is more straightforward than copying the array.